data/distro.yml

```diff
@@ -49,6 +49,25 @@
         </code></pre>
       </p>
     </ol>
+    
+- name: "Manjaro"
+  logo: "manjaro.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Enable Flatpak through the Software Manager</h2>
+        <p>Flatpak is installed by default on Manjaro 20 or higher.</p>
+        <p>To enable its support, navigate to the <strong>Software Manager</strong>(Add/Remove Programs)</p>
+        <p>Click on the triple line menu [or dots depending on the Desktop Environment] on the right, in the drop down menu select "Preferences"</p>
+        <p>Navigate to the tab "Flatpak" and slide the toggle to Enable Flatpak support (It is also possible to enable checking for updates, which is recommended).</p>
+        <p>Note: The installation of the Flathub repository is automatic on Manjaro.</p>
+        <p>Now you just need to exit and restart your system to get Flatpak support through the Software Manager.</p>
```

Comment:
This is not needed as well. It is already mentioned in the "Restart" section.

Reply:
Okay, will fix these, any other issues you noticed before I do?

Reply:
It looks fine otherwise.